### PR TITLE
Render "message" as final crumb when no exception

### PR DESCRIPTION
**Pull request by benvinegar**
We were only rendering "exception" events as breadcrumbs. This adds "message" events (no exception attached).

image

/cc @getsentry/ui

### CODE REVIEW

**Comment by benvinegar** [comment]
Just noticed this needs a "final" class or something so that the timeline stops on the final crumb rather than pass through.

**Comment by mitsuhiko** [comment]
What does that look like with humongous messages?

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs.jsx [hunk 1]
```diff
@@ -70,6 +70,36 @@ const BreadcrumbsInterface = React.createClass({
...
+      let levelTag = evt.tags.find(t => t.key === 'level');
+      let level = levelTag && levelTag.value;
+      crumb = {
+        type: 'message',
+        level: level,
+        category: 'message',
+        message: evt.message
+      };
+    }
+    crumb.timestamp = evt.dateCreated;
```
Comment (macqueen) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs.jsx]: it seems like it's possible for `crumb` to be undefined, so we should probably check for that before trying to set `crumb.timestamp`

**Comment by macqueen** [comment]
👍

**Comment by benvinegar** [comment]
@mitsuhiko 

image

This is the same as if there was a long exception string.

**Comment by benvinegar** [comment]
Okay, I just made a few changes if people want to take a second look (sorry)!

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumb.jsx [hunk 2]
```diff
@@ -27,6 +27,10 @@ const Breadcrumb = React.createClass({
     if (crumb.category && crumb.category.slice(0, 3) === 'ui.') {
       rv += ' crumb-user';
     }
+
+    if (crumb.last) {
+      rv += ' crumb-last';
```
Comment (ckj) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumb.jsx]: 🆒

**Comment by ckj** [comment]
👍  from a UI standpoint.

**Pull request merged**